### DISCUSSION TITLE
Why would i want to connect to other nodes?

### DISCUSSION THREAD

**Discussion by GeorgePapageorgakis** [Chia-Network/chia-blockchain#1436] [General]
Why would I want to connect to specific nodes from the "Connect to other peers"? can we enable some kind of "pool" farming?

**Comment by hoffmang9** [comment]
A peer to peer network can not work without connecting to other peers.

Pools coming - https://www.chia.net/2020/11/10/pools-in-chia.html

**Reply by GeorgePapageorgakis** [reply]
My mistake for the misleading question. I have updated it.
I mean, is there any benefit if I want to connecto specific peers? I guess its useful when I have remote nodes or so?

**Reply by dorkmo** [reply]
ive used it to connect to known peers on one machine, to speed up discovery on a new machine

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
